features/support/hooks.el

```diff
@@ -0,0 +1,12 @@
+(require 'ecukes)
+
+(defun kill-js-buffers ()
+  (mapcar (lambda (buffer)
+            (when (string-match "\.js" (buffer-name buffer))
```

Comment:
Updated. Is using `or` idiomatic for getting a default string value? I am still pretty new to elisp